utils/core-contracts/contracts/interfaces/IERC20Permit.sol

```diff
@@ -0,0 +1,60 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./IERC20.sol";
+
+/**
+ * @dev Interface of the ERC20 Permit extension allowing approvals to be made via signatures, as defined in
```

Comment:
Please use @title here, and @notice in user facing functions. It's what we're doing elsewhere when decoumenting.